### DISCUSSION TITLE
DAX Studio connection mechanism to semantic models

### DISCUSSION THREAD

**Discussion by MrKarol** [DaxStudio/DaxStudio#1252] [Q&A] [answered]
Hi All,

Could you share what type of a connection mechanism (protocols, authetication and authorization types) DAX Studio uses to connect to local and Power BI service semantic models? Also, any links to DAX Studio architecture diagram would be very useful. 

Thank you
K

**Comment by dgosbell** [comment, marked as answer]
We should have most of that covered here https://daxstudio.org/docs/security-privacy/
